drivers/spi/spi_dw.c

```diff
@@ -317,23 +320,31 @@ static int transceive(struct spi_config *config,
 }
 
 static int spi_dw_transceive(struct spi_config *config,
-			     const struct spi_buf **tx_bufs,
-			     struct spi_buf **rx_bufs)
+			     const struct spi_buf *tx_bufs,
+			     size_t tx_count,
+			     struct spi_buf *rx_bufs,
+			     size_t rx_count)
 {
-	SYS_LOG_DBG("%p, %p, %p", config->dev, tx_bufs, rx_bufs);
+	SYS_LOG_DBG("%p, %p (%d), %p (%d)",
+		    config->dev, tx_bufs, tx_count, rx_bufs, rx_count);
```

Comment:
Correct format specifier for size_t is %zu (otherwise you risk compiler warnings)